editors/vim/syntax/lambdapi.vim

```diff
@@ -127,7 +127,7 @@ syntax match Keyword "@"
 syntax match Keyword "\["
 syntax match Keyword "\]"
 syntax match Keyword ":"
-syntax match Keyword "⇒"
+syntax match Keyword "↪"
 syntax match Keyword "→"
 syntax match Keyword "∀"
```

Comment:
∀ is not a LP keyword, nor [ and ]. On the other hand are missing: ↪ , Π, ≔

Reply:
Well I think we can remove parentheses as well.